src/System.Private.Xml/src/System/Xml/IApplicationResourceStreamResolver.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.Xml
 {
-    [Obsolete("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)]
+    [Obsolete("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", false)]
```

Comment:
nit: add a comment here explaining the reason this differs from the reference assembly. EG: We must specify error=false in the implementation assembly so that we can typeforward to this type without hitting the compiler error.